src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -123,6 +126,110 @@ public class ProtoCompile : ToolTask
                                                         "javanano", "js", "objc",
                                                         "php", "python", "ruby" };
 
+        static readonly TimeSpan RegexTimeout = TimeSpan.FromMilliseconds(100);
```

Comment:
Should these new fields follow the `s_` naming standard from `s_supportedGenerators`